tests/testthat/test-network.R

```diff
@@ -147,13 +147,16 @@ test_that("We can remove links by weight", {
 test_that("toJSON works for networks", {
   # Create some nodes
   nodeA <- Node(
-    id = NodeId('A')
+    id = NodeId('A'),
+    degree = 2
   )
   nodeB <- Node(
-    id = NodeId('B')
+    id = NodeId('B'),
+    degree = 2
   )
   nodeC <- Node(
-    id = NodeId('C')
+    id = NodeId('C'),
+    degree = 2
```

Comment:
i dont think the degree should allowed to be set when creating a node. The degree is not a thing outside of the network context. First, I could have nodeA in network1 and network2. There's no reason nodeA should have the same degree in both. 
Second, i could set it wrong. Why would i know that nodeA will have two links when i haven't made the edge list yet? These degrees aren't connected to the edge list at all

Reply:
what if i just dont export it from the package, and it can only ever be used internally? i dont see why, if you have an edgeList in hand, you shouldnt be able to create the node already knowing the degree. but i agree random people shouldnt make random nodes w random values for degree.

Reply:
i think id even be happy if the only thing we actually did export were the edgeList based constructors, particularly for now